modules/user.definitions.yaml

```diff
@@ -540,3 +540,24 @@ json_tripgouser:
     the backend, cannot be used to updated
 
     those lists, for that, you need to use the endpoints for that purpose.
+json_push_device:
+  type: object
+  title: PushNotificationDevice
+  properties:
+    platform:
+      type: string
+      enum:
+        - android
+        - ios
+      description: android or ios device
+    deviceid:
+      type: string
+      description: a unique id for that user/device pair
```

Comment:
Is this a token that the API user makes up or is there some standard way for getting such a token for Android and iOS respectively?